dictionaries/CallMap_84_delta.php

```diff
@@ -0,0 +1,1733 @@
+<?php // phpcs:ignoreFile
+namespace Phan\Language\Internal;
+
+return array (
```

Comment:
Why `array`?? Other files just uses `[`
https://github.com/vimeo/psalm/blob/bfbd53c48ec0853308c1758dd80e512a0b8558d9/dictionaries/CallMap_83_delta.php#L18-L19

Also PHP Docblock
https://github.com/vimeo/psalm/blob/bfbd53c48ec0853308c1758dd80e512a0b8558d9/dictionaries/CallMap_83_delta.php#L16

Reply:
Both are not relevant, the files will now be auto generated